force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -62,14 +68,27 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Boolean> checkStringContent(String input) {
         // Initialize a list to store the results
-        
+        List<Boolean> booleanList = new List<Boolean>();
         // Check if the string starts with 'Sales'
-        
+        if (input.startsWith('Sales')) {
+            booleanList.add(true);
+        } else {
+            booleanList.add(false);
+        }
         // Check if the string ends with 'Force'
-        
+        if (input.endsWith('Force')) {
+            booleanList.add(true);
+        } else {
+            booleanList.add(false);
+        }
         // Check if the string contains 'Awesome'
-        
-        return null; // Replace null with the variable you used to store the result
+        if (input.contains('Awesome')) {
+            booleanList.add(true);
+        } else {
+            booleanList.add(false);
+        }
+
+        return booleanList; 
```

Comment:
You made this more complicated for yourself that it needed to be. Think about what each expression your write will evaluate to, for example input.contains().. returns a boolean. No need to have any if statements here.